packages/amplify_datastore/example/ios/unit_tests/DataStoreHubEventStreamHandlerTests.swift

```diff
@@ -63,7 +63,7 @@ class DataStoreHubEventStreamHandlerTests: XCTestCase {
         waitForExpectations(timeout: 1.0)
```

Comment:
Not related to this PR, but we should remove these from generated files in `.gitattributes`. Everything under `**/example/ios/**` is treated as generated code since the majority of it is generated by flutter during `flutter create`. We should remove `unit_tests` from this.

Could you make that change in this feature branch (can be in a follow up PR) since there will probably be quite a few PRs that will updates native tests?

Reply:
Good call out, I added that to this branch.

Reply:
Thank you